ios/Runner/Info.plist

```diff
@@ -43,5 +43,9 @@
 	<false/>
 	<key>CADisableMinimumFrameDurationOnPhone</key>
 	<true/>
+	<key>NSPhotoLibraryAddUsageDescription</key>
+    <string>FLogs would like to save photos from the app to your gallery</string>
+    <key>NSPhotoLibraryUsageDescription</key>
+    <string>FLogs would like to access your photo gallery for uploading images to the app</string>
```

Comment:
🤔 this might sound a bit fishy to the users, but indeed this is the way they recommend to use this package. I guess it just as good as anything.

Reply:
❓If the user does not give the permission, the app still works right? (I hate it when apps require certain permissions and otherwise don't work even though the permissions are not actually essential for using the app...)

We could try to add a bit more context so the user understands better why this is needed and what `FLogs` is. 
Do we even have to mention `FLogs` or can we just state something along the lines of:

```
In order to save log information and debug data the app would like to save photos from the app to your gallery.
```

Why is the other direction needed - I get that we need rights to save images, but why also uploading rights?

Reply:
We could also try a different avenue - AFAIK there's no need for any of these permissions on iOS if you only use the 'send logs to the devs' feature in an event of a crash. did that in testflight and app I was beta testing didn't need any permissions.

Reply:
> 🤔 this might sound a bit fishy to the users, but indeed this is the way they recommend to use this package. I guess it just as good as anything.

I just tested it on the iphone simulator and FLog.exportLogs() worked with and without the permissions set. I guess it needs to be tested on a real device?

However, I looked over the [docs](https://pub.dev/packages/f_logs) and there are also other ways of exporting the logs - no need to decide now.

I am going to remove the permissions for now and leave that decision up for https://github.com/itchysats/10101/issues/105